data-raw/plot_data.R

```diff
@@ -0,0 +1,88 @@
+library(readxl)
+
+#Uploading data into R
+plot_data01 <- read_excel("data-raw/MacLeish Data_Plot MC-01.xlsx")
+plot_data02 <- read_excel("data-raw/MacLeish Data_Plot MC-02.xlsx")
+
+usethis::use_data(plot_data01, overwrite = TRUE)
+usethis::use_data(plot_data02, overwrite = TRUE)
+
+library(tidyverse)
+library(janitor)
+library(lubridate)
+library(stringr)
+library(usethis)
+library(devtools)
+library(dplyr)
+
+#Plot Data 1
+#Creating year and height column 
+newplot_data01 <- plot_data01 %>%
+  pivot_longer(c(`Fall'15`, `Fall'14`,`Fall'13`,`Fall'12`,`Fall'11`,`Fall'10`), names_to = "Year", values_to = "Diameter(cm)")
+
+#Separating fall and year into new columns 
+newplot_data01 <- newplot_data01 %>%
+  separate(Year, c("Semester", "Year"), sep = 4)
+
+#Changing abbreviated year into full year 
+newplot_data01$Year[newplot_data01$Year == "'15"] <- "2015"
+newplot_data01$Year[newplot_data01$Year == "'14"] <- "2014"
+newplot_data01$Year[newplot_data01$Year == "'13"] <- "2013"
+newplot_data01$Year[newplot_data01$Year == "'12"] <- "2012"
+newplot_data01$Year[newplot_data01$Year == "'11"] <- "2011"
+newplot_data01$Year[newplot_data01$Year == "'10"] <- "2010"
+
+#Changing value "DEAD" to NA for diameter
+#Tree diameter didn't increase or decrease over time 
+newplot_data01$`Diameter(cm)`[newplot_data01$`Diameter(cm)` == "DEAD"] <- "NA"
+
+#Shifting the "Notes:" column to the end of the dataset 
+newplot_data01 <- newplot_data01 %>%
+  relocate("Notes:", .after = "Diameter(cm)")
+
+View(newplot_data01)
+
+plot_data01 <- newplot_data01
+
+usethis::use_data(plot_data01, overwrite = TRUE)
+
+
+#Plot Data 2
+library(readxl)
```

Comment:
Basic principle of good coding: [DRY](https://en.wikipedia.org/wiki/Don%27t_repeat_yourself) Don't repeat yourself.  You already loaded package above